app/models/enterprise.rb

```diff
@@ -127,6 +127,9 @@ class Enterprise < ApplicationRecord
               message: Spree.t('errors.messages.invalid_instagram_url')
             }, allow_blank: true
   validate :validate_white_label_logo_link
+  validates :external_billing_id,
+            format: { with: /\A\S+\Z/ },
```

Comment:
I wasn't familiar with the `\A` or `\Z`, so I tested this out. A website like [regexr.com](https://regexr.com/) is really handy for testing and perfecting regular expressions.

I don't think this is what was intended, and it would always be considered invalid. Can you please add at least one other test to ensure that a valid Billing ID is validated correctly.

![Screenshot 2025-01-28 at 12 47 40 pm](https://github.com/user-attachments/assets/47b52695-0525-4cde-b4af-3ed51cae2caf)

Reply:
Yes, it seems I messed up with the regex, not sure how the tests is passing... I will update with a standard one : `/^\S+$/`.
![image](https://github.com/user-attachments/assets/4380aafd-c935-462a-9221-4c02b76d4e55)